template.rb

```diff
@@ -8,9 +8,10 @@
 # This template assumes you have followed the setup guide at:
 # https://github.com/wildland/guides/#setting-up-your-development-enviroment
 #
-ruby_version = '2.2.2'
-node_version = 'v4.0.0'
+ruby_version = '2.2.3'
+node_version = 'v5.4.0'
 action_messages = []
+branch = 'update-template-to-current'
```

Comment:
Why was this changed from master? Also I like the removal of the magic number and moving this to a variable.